### PR TITLE
Added two file extensions popularized by other software:

### PR DESCRIPTION
**Pull request by Nusiq**
- animation.json - used by Blockbench for resource pack animations
- entity.json - used in default Minecraft resource packs for client entities

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -566,7 +566,7 @@
           "*Resource*Pack*/animations/**/*.{json,jsonc,json5}",
           "*RP*/animations/**/*.{json,jsonc,json5}",
           "*rp*/animations/**/*.{json,jsonc,json5}",
-          "*.{animation.rp,animation,anim.rp,a.rp,rpa}.{json,jsonc,json5}"
+          "*.{animation.rp,animation,anim.rp,a.rp,rpa,animation}.{json,jsonc,json5}"
```
Comment (DaanV2) [inline package.json:569]: animation is already there, item nr 2 😛 
I added it when you mentioned it couple of days ago 😄

Comment (Nusiq) [reply package.json:569]: I'm blind :(

Comment (DaanV2) [reply package.json:569]: Nw 😄

**Comment by Nusiq** [comment]
I undone the changes created by the bot (by force pushing). And removed the duplicate `animation` extension.

**Pull request merged**